scene/resources/resource_format_text.cpp

```diff
@@ -927,11 +931,24 @@ Error ResourceLoaderText::rename_dependencies(Ref<FileAccess> p_f, const String
 		} else {
 			if (fw.is_null()) {
 				fw = FileAccess::open(p_path + ".depren", FileAccess::WRITE);
-				if (is_scene) {
-					fw->store_line("[gd_scene load_steps=" + itos(resources_total) + " format=" + itos(FORMAT_VERSION) + "]\n");
-				} else {
-					fw->store_line("[gd_resource type=\"" + res_type + "\" load_steps=" + itos(resources_total) + " format=" + itos(FORMAT_VERSION) + "]\n");
+
+				// Copy the title tag.
+				uint64_t f_position = f->get_position();
+				f->seek(0);
+				uint64_t num_remaining = tag_end;
+
+				num_read = f->get_buffer(buffer, buffer_size);
+				ERR_FAIL_COND_V_MSG(num_read == UINT32_MAX, ERR_CANT_CREATE, "Failed to allocate memory for buffer.");
+
+				while (num_read < num_remaining) {
+					fw->store_buffer(buffer, num_read);
+					num_remaining -= num_read;
+					num_read = f->get_buffer(buffer, buffer_size);
 				}
+				fw->store_buffer(buffer, num_remaining);
+				fw->store_line("\n");
+				// Prevent errors when reusing the stream.
+				f->seek(f_position);
 			}
```

Comment:
This section of code appears to do a different thing to the original code. Although it may copy verbatim now, it may change in future (say if e.g. `FORMAT_VERSION` changes). I would leave this section as is to be honest.

Reply:
I'm not particularly familiar with this one. It feels inappropriate to only change the title if `FORMAT_VERSION` changes, and subsequent tags may also need to be updated according to the upgraded version format. But the subsequent ones are copied as-is as well.

Reply:
Well, it really won't be 0, but the reason I gave earlier was wrong. It is indeed related to re-seek, the code here returns to the end of the penultimate tag. There is at least one tag text waiting to be copied.